src/ddmd/parse.d

```diff
@@ -21,6 +21,7 @@ import ddmd.root.filename;
 import ddmd.root.outbuffer;
 import ddmd.root.rmem;
 import ddmd.root.rootobject;
+import ddmd.target;
```

Comment:
No, I mean that the fix was done two days ago, and should have gone in yesterday.